commands/password-managers/bitwarden/copy-first-matching-password.sh

```diff
@@ -20,6 +20,10 @@
 # @raycast.author Phil Salant
 # @raycast.authorURL https://github.com/PSalant726
 # @raycast.description Search all items in a Bitwarden vault, and copy the password of the first search result to the clipboard.
+notFound() {
+  echo -n "The query '$1' did not return a password."
```

Comment:
Not a big deal, but since the next line will always exit the script, `-n` is unnecessary and just adds extra complexity.
```suggestion
  echo "The query '$1' did not return a password."
```